ringpop.go

```diff
@@ -226,12 +258,23 @@ type BootstrapOptions struct {
 
 // Bootstrap starts the Ringpop
 func (rp *Ringpop) Bootstrap(opts *BootstrapOptions) ([]string, error) {
+
```

Comment:
Yep, already did it:
https://github.com/uber/ringpop-go/commit/587528f5811bb555c549abb0c6ea23513a7d472b